src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -1,30 +1,258 @@
-﻿<Project Sdk="MSBuild.Sdk.Extras">
+﻿<Project  Sdk="MSBuild.Sdk.Extras">
```

Comment:
No need to review this except the new flags SUPPORTS_CUSTOM_CACHE etc. 
It was just copied from directory.build.props